### PR TITLE
[NT-203] (2/2) Pledge summary section

### PR DESCRIPTION
**Pull request by Scollaco**
# 📲 What

- This is the second part of a 2-part work to build the Pledge summary section of the  Manage Pledge screen.

# 🤔 Why

Because the work has to be done 😎 .
 
# 🛠 How

- Added a new property `locationName` to the Backing model.
- Updated strings
- Added missing tests

# 👀 See
It shouldn't have difference between the work done here and and this [PR](https://github.com/kickstarter/ios-oss/pull/845#issue-318485091).

### CODE REVIEW

File: KsApi/models/Backing.swift [hunk 1]
```diff
@@ -9,6 +9,7 @@ public struct Backing {
   public let backerCompleted: Bool?
   public let id: Int
   public let locationId: Int?
+  public let locationName: String?
```
Comment (justinswart) [inline KsApi/models/Backing.swift]: Just wondering if we prefer this over just passing the `Reward` to this VC? That already has all of the shipping info and would save us a lot of code.

**Comment by Scollaco** [comment]
> Hey I have the one question and also wondering if we could rename `ManageViewPledgeViewController` -> `ManagePledgeViewController` in this PR?

Yeah, since the PR is small and simple, could be a good opportunity to rename it

File: KsApi/models/Backing.swift (hunk 1 above)
Comment (justinswart) [reply KsApi/models/Backing.swift]: Cool and what about this? 😄

Comment (justinswart) [reply KsApi/models/Backing.swift]: We chatted on Slack 👍

Comment (Scollaco) [reply KsApi/models/Backing.swift]: From Slack: ` The reward contains shipping rules, but it doesn’t contain the shipping rule you chose`.

**Pull request merged**